draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +920,37 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> First generations of quantum networks will not be store-and-forward
+           networks.
+           <vspace blankLines="1" />
+
+           As described in <xref target="elg"></xref>,
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed links of classical networks.
+           This phenomenological distinction should result in architectural
+           differences between quantum networks and classical networks;
+           quantum networks put link Bell pairs together to an end-to-end
+           Bell pair where classical networks deliver messages from an end
+           to the other end hop by hop.
+           <vspace blankLines="1" />
+
+           Classical networks store data coming from an interface, then forward
+           the data to another appropreate interface.
+           Quantum networks store a couple of half Bell pairs, then
+           execute entanglement swapping instead of forwarding in a data plane.
+           Such quantum networks are "store and swap" networks.
+           In "store and swap" networks, we do not need to care about
+           the order of generating link Bell pairs since Bell pairs are
+           undirected resources. This distinction makes control algorithms and
+           optimization of quantum networks different from classical ones.
+           <vspace blankLines="1" />
+
+           Meanwhile, third generations of quantum networks of direct transmission
+           are "store and forward" networks, since such quantum networks can carry
+           arbitrary states hop by hop. Hence those later generations of
+           quantum networks would have more in common with classical networks
+           than the first quantum networks in networking algorithms.</t>
+           
```

Comment:
```suggestion
           It is worth noting, that third generation <refer to generations section> 
           quantum repeaters will be able to transmit qubits directly
           in a "store and forward" manner since they will be able to carry
           arbitrary states hop by hop. Thus, the networking algorithms of the
           later generations of quantum networks may end up having more in 
           common with classical networks.</t>
           
```